contracts/colonyNetwork/IColonyNetwork.sol

```diff
@@ -195,13 +195,20 @@ contract IColonyNetwork is ColonyNetworkDataTypes, IRecovery {
   /// @return resolverAddress Address of the `Resolver` contract
   function getColonyVersionResolver(uint256 _version) public view returns (address resolverAddress);
 
-  /// @notice Set a new Reputation root hash and starts a new mining cycle. Can only be called by the ReputationMiningCycle contract.
+  /// @notice This version of setReputationRootHash is deprecated and will be removed in a future release. It transparently
+  /// calls the new version if it is called (essentially, removing the `reward` parameter.
   /// @param newHash The reputation root hash
   /// @param newNLeaves The updated leaves count value
   /// @param stakers Array of users who submitted or backed the hash, being accepted here as the new reputation root hash
-  /// @param reward Amount of CLNY to be distributed as reward to miners
+  /// @param reward Amount of CLNY to be distributed as reward to miners (not used)
   function setReputationRootHash(bytes32 newHash, uint256 newNLeaves, address[] memory stakers, uint256 reward) public;
 
+  /// @notice Set a new Reputation root hash and starts a new mining cycle. Can only be called by the ReputationMiningCycle contract.
+  /// @param newHash The reputation root hash
+  /// @param newNLeaves The updated leaves count value
+  /// @param stakers Array of users who submitted or backed the hash, being accepted here as the new reputation root hash
+  function setReputationRootHash(bytes32 newHash, uint256 newNLeaves, address[] memory stakers) public;
```

Comment:
Is this new?

Reply:
Yep, the original (deprecated in this PR) has the `reward` parameter, which at some point in the past I thought would come from the `ReputationMiningCycle`. We have to keep both for transition purposes (as whenever we deploy, the already-deployed ReputationMiningCycle contracts will expect to be able to call the one with the `reward` parameter). The deprecated one acts as a passthrough for this one.